api.go

```diff
@@ -234,3 +240,9 @@ func buildURL(path string) string {
 
 	return fmt.Sprintf("http://%s%s", u, path)
 }
+
+func checkError(err error) {
+	if err != nil {
+		log.Fatal(err)
+	}
+}
```

Comment:
@Asakiz I am not against the logging but we need to return on the API if it succeed or error out.